crates/solidity/outputs/rust/lib/src/generated/kinds.rs

```diff
@@ -5,107 +5,114 @@ use strum_macros::*;
 
 #[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord, Serialize)]
 pub enum TokenKind {
-    Abicoder,
-    Abstract,
+    AbicoderKeyword,
+    AbstractKeyword,
     Address,
+    AddressKeyword,
     Ampersand,
     AmpersandAmpersand,
     AmpersandEqual,
-    Anonymous,
-    As,
+    AnonymousKeyword,
+    AsKeyword,
     AsciiEscape,
     AsciiStringLiteral,
-    Assembly,
+    AssemblyKeyword,
     Bang,
     BangEqual,
-    Bool,
-    BooleanLiteral,
-    Break,
-    Calldata,
+    BoolKeyword,
+    BreakKeyword,
+    CalldataKeyword,
     Caret,
     CaretEqual,
-    Case,
-    Catch,
+    CaseKeyword,
+    CatchKeyword,
     CloseBrace,
     CloseBracket,
     CloseParen,
     Colon,
     ColonEqual,
     Comma,
-    Constant,
-    Constructor,
-    Continue,
-    Contract,
+    ConstantKeyword,
+    ConstructorKeyword,
+    ContinueKeyword,
+    ContractKeyword,
+    DaysKeyword,
     DecimalExponent,
     DecimalLiteral,
     DecimalNumber,
-    Default,
-    Delete,
-    Do,
+    DefaultKeyword,
+    DeleteKeyword,
+    DoKeyword,
     DoubleQuoteEvmasmDoubleQuote,
     DoubleQuotedAsciiStringLiteral,
     DoubleQuotedUnicodeStringLiteral,
-    Else,
-    Emit,
+    ElseKeyword,
+    EmitKeyword,
     EndOfLine,
-    Enum,
+    EnumKeyword,
     Equal,
     EqualEqual,
     EqualGreater,
-    Error,
+    ErrorKeyword,
     EscapeSequence,
-    Event,
-    Experimental,
-    External,
+    EtherKeyword,
+    EventKeyword,
+    ExperimentalKeyword,
+    ExternalKeyword,
     Fallback,
```

Comment:
I think we missed a `Fallback` in `crates/solidity/inputs/schema/grammar/topics/02-definitions/08-functions.yml`

Reply:
done